README.md

```diff
@@ -42,11 +42,19 @@ script/{your-framework}/start
 
 This will get you the foo site running at http://127.0.0.1:8008, the bar interface running at http://127.0.0.1:8009 and a webserver running at http://127.0.0.1:3000.
 
-## Task
+## Your set-up task
 1. Run Foo at http://127.0.0.1:8008, verify that an alert with `Yay! Everything works` fires
 2. Run Bar at http://127.0.0.1:8009, verify that an alert with `Yay! Everything works` fires
 
-## Structure
+🥳 If those worked, well done! You're all set for the interview! 
```

Comment:
Making it clear that they don't need to do anything else for the set-up - can someone confirm that's correct?